uamqp/async_ops/client_async.py

```diff
@@ -945,7 +950,7 @@ def receive_messages_iter_async(self, on_message_received=None):
         :rtype: Generator[~uamqp.message.Message]
         """
         self._message_received_callback = on_message_received
-        self._received_messages = queue.Queue()
+        self._received_messages = self._received_messages or compat.queue.Queue()
```

Comment:
Is this assignment statement still necessary? `self._received_messages` should never be `None` now right?